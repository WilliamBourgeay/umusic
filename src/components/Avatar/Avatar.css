@media (max-width: 600px) {
  .avatar {
    display: inline-flex;
    justify-content: center;
    width: 150px !important;
    height: 150px !important;
    position: relative;
    z-index: 1;
  }

  .displayFlex {
    display: flex;
    padding: 16px;
    justify-content: center;
  }
}

@media (min-width: 600px) {
  .avatar {
    display: inline-flex;
    margin: -75px auto 0 auto;
    justify-content: center;
    width: 150px !important;
    height: 150px !important;
    position: relative;
    z-index: 1;
  }

  .displayFlex {
    display: flex;
  }
}

.hoverAvatar {
  cursor: pointer;
  padding: 8px;
}

.friendAvatar {
  display: flex;
  margin-top: 3%;
  margin-right: 5%;
  margin-left: 5%;
  justify-content: center;
  width: 50px !important;
  height: 50px !important;
}
