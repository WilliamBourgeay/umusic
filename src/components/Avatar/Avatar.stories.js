import React from 'react';
import { storiesOf } from '@storybook/react';
import { MyAvatar } from './Avatar';

const styledUser = { borderRadius: '100px', width: '100px', height: '100px' };

const stories = storiesOf('Avatar', module);

stories.add('Avatar', () => (
  <MyAvatar email="wawrzyniakjulien@gmail.com" hover="displayFlex" styled={styledUser} />
));
