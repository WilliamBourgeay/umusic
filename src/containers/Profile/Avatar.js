/** *************************************
** O-rizon development
** Created by Alexy Hostetter
** 26/10/18 - 21:07
** Avatar.js
** 2018 - All rights reserved
************************************** */

// @flow

import React from 'react';
import Avatar from '@material-ui/core/Avatar';
import './Profile.css';

type Props = {
  cN: string,
  user: string,
  img: string,
}

const MyAvatar = ({ cN, user, img }: Props) => (
  <Avatar
    className={cN}
    alt={user}
    src={img}
  />
);

export default MyAvatar;
