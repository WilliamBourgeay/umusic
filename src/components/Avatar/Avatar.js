// @flow

import React from 'react';
import Avatar from '@material-ui/core/Avatar';
import '../../containers/Profile/Profile.css';

type Props = {
  cN: string,
  user: string,
  img: string,
}

const MyAvatar = ({ cN, user, img }: Props) => (
  <Avatar
    className={cN}
    alt={user}
    src={img}
  />
);

export default MyAvatar;
