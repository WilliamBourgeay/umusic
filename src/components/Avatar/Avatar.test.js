import * as React from 'react';
import renderer from 'react-test-renderer';

import { MyAvatar } from './Avatar';

describe('MyAvatar', () => {
  it('should render a MyAvatar', () => {
    const tree = renderer.create(
      <MyAvatar email="wawrzyniakjulien@gmail.com" hover="displayFlex" styled={{}} />,
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
