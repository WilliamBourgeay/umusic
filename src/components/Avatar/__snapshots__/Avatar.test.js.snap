// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyAvatar should render a MyAvatar 1`] = `
<div
  className="displayFlex"
  onClick={[Function]}
  role="presentation"
>
  <img
    alt="Gravatar for wawrzyniakjulien@gmail.com"
    className="react-gravatar"
    height={50}
    src="//www.gravatar.com/avatar/24ee74d518b120620baa1f722b5edc0d?d=retro&r=g&s=50"
    srcSet="//www.gravatar.com/avatar/24ee74d518b120620baa1f722b5edc0d?d=retro&r=g&s=100 2x"
    style={Object {}}
    width={50}
  />
</div>
`;
